packages/TorneloScoresheet/src/components/GraphicalModePlayerCard/GraphicalModePlayerCard.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { View } from 'react-native';
+import { colours } from '../../style/colour';
+import { Player } from '../../types/ChessGameInfo';
+import PrimaryText, { FontWeight } from '../PrimaryText/PrimaryText';
+import { styles } from './style';
+
+type recordingPlayerCardProps = {
+  player: Player;
+  align: 'left' | 'right';
+};
+
+const GraphicalModePlayerCard: React.FC<recordingPlayerCardProps> = ({
+  player,
+  align,
+}) => {
+  const flexStyle = align == 'left' ? styles.flexRow : styles.flexRowReverse;
+  const cardStyle = { ...styles.graphicalModePlayerCard, ...flexStyle };
+
+  const teamNameAndEloTextSection = (label: string) => (
+    <PrimaryText
+      size={15}
+      weight={FontWeight.Medium}
+      colour={colours.secondary70}
+      label={label}
+    />
+  );
```

Comment:
Let's just make this a component? I can still be in this file, but it's a bit strange to make it a function when it is playing the role of a component

Reply:
Refactored to `CardTextRowComponent`